Translations/pl.po

```diff
@@ -4631,7 +4631,7 @@ msgstr "menu"
 
 #: Source/panels/mainpanel.cpp:91
 msgid "inv"
-msgstr "ekwipunek"
+msgstr "inw"
```

Comment:
Why "inw", where it comes from? "Inwentarz"? If yes then it is inconsistent -- in other places in this file "inventory" is translated to "ekwipunek", using a shortened different word just in this context looks strange.

Reply:
I believe shortened form/abbreviation is needed to fit on the button and look decent.

Reply:
"ekwipunek" does fit, doesn't look the best, but readable:

![20240117195412_1](https://github.com/diasurgical/devilutionX/assets/896399/5d29d53d-c174-4111-a5eb-902d2e55b0e3)

"inw" would look strange to be honest.

Reply:
English version uses "Inv" instead of "Inventory". For the same reason Polish version should use "Ekw" (or maybe "Ekwip"?) instead of "Ekwipunek". Honestly I hate how Ekwipunek button looks, letters are super tight

Reply:
> For the same reason

I don't think Polish necessarily has the game linguistic rules as English.

Reply:
Zasoby, Zapasy, Skarby

Reply:
> > For the same reason
> 
> I don't think Polish necessarily has the game linguistic rules as English.

They shortened english Inventory into Inv so I don't see a reason not to shorten polish Ekwipunek. It has nothing to do with linguistic rules. Just saying shortening is not a crime if the made it in official release... Also I am Polish if that matters.

Reply:
> > > For the same reason
> > 
> > 
> > I don't think Polish necessarily has the game linguistic rules as English.
> 
> They shortened english Inventory into Inv so I don't see a reason not to shorten polish Ekwipunek. It has nothing to do with linguistic rules. Just saying shortening is not a crime if the made it in official release...

Inventory -> inv -- makes sense, Ekwipunek -> inw -- what? If anything "Ekw" or "Ekwip", but both look a bit strange, I'm just saying there's no good short version for "Ekwipunek".

> Also I am Polish if that matters.

Yeah, I know the pain man, same here ;)